File: PersonalFitnessCare.Server/Controllers/HomeController.cs
namespace PersonalFitnessCare.Server.Controllers
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Web;
    using System.Web.Mvc;
    //using System.Web.Http.Cors;

    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewBag.Title = "Home Page";

            return View();
        }
    }
}

File: PersonalFitnessCare.Client/scripts/app/home/home-controller.js
(function () {
    'use strict';

    function HomeController(people) {
        var vm = this;

        people.search({ page: 1 })
            .then(function (initialCats) {
                vm.people = initialCats;
            });
    }

    angular.module('fitnessApp.controllers')
        .controller('HomeController', ['people', HomeController]);
}());